templates/vue/src/components/lightbox/H5PIframe.vue

```diff
@@ -203,7 +201,7 @@ export default {
         })
       }
       $(window).resize(setIframeDimensions)
-      setIframeDimensions()
+      setIframeDimensions() */
```

Comment:
I'm not super sure what this block of code does. I commented it out because it was affecting the positioning of the scrollbar, and there didn't seem to be any differences.

Reply:
This was added in to automatically size the iframe for the H5P to fit into the lightbox. Are you sure it works without it?

Reply:
Did some testing and can confirm that it works properly.